backend/src/main/scala/akka/viz/server/Webservice.scala

```diff
@@ -27,8 +28,8 @@ object ApiMessages {
 
 }
 
-class Webservice(implicit fm: Materializer, system: ActorSystem) {
-
+class Webservice(implicit fm: Materializer, system: ActorSystem) extends SubscriptionSession {
+  import Helpers.actorRefToString
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2004142104/issues/source?bid=3121282&fileBranchId=3122179#l32)